.semaphore/semaphore.yml

```diff
@@ -19,51 +19,53 @@ blocks:
             - cache restore
             - yarn install
             - cache store
-
-  - name: Lint
-    dependencies: ["Install dependencies"]
-    # Run this block on changes in frontend folder
+  - name: Test
+    dependencies: []
     run:
       when: "change_in('/frontend')"
     task:
       jobs:
-        - name: lint client
+        - name: test client
           commands:
             - checkout
             - cd frontend
             - cache restore
-            - yarn lint
-
-  - name: Build
+            - yarn install
```

Comment:
should remove it?

Reply:
@andersonmalheiro add this line
```
dependencies: ["Install dependencies"]
```
then we can remove the install command. 👍